src/modules/pos/validator/components/ValidatorsMonitorView/Validators.js

```diff
@@ -16,17 +16,21 @@ import BoxTabs from '@theme/tabs';
 import useFilter from '@common/hooks/useFilter';
 import DialogLink from '@theme/dialog/link';
 import { useCurrentAccount } from '@account/hooks';
-import InfoBanner from '@common/components/infoBanner/infoBanner';
 import Icon from '@theme/Icon';
+import { INFO_BANNERS } from '@common/constants';
 import { ROUND_LENGTH } from '@pos/validator/consts';
 import { PrimaryButton, SecondaryButton } from '@theme/buttons';
 import { useBlocks } from '@block/hooks/queries/useBlocks';
+import SwippableInfoBanner from '@common/components/infoBanner/swippableInfoBanner';
 import ValidatorsOverview from '../Overview/ValidatorsOverview';
 import GeneratingDetails from '../Overview/GeneratingDetails';
 import ValidatorsTable from '../ValidatorsTable';
 import LatestStakes from '../LatestStakes';
 import { useValidators } from '../../hooks/queries';
 import styles from './Validators.css';
+import 'swiper/css';
+import 'swiper/css/pagination';
+import 'swiper/css/navigation';
```

Comment:
Swiper styles are already imported in SwippableInfoBanner component and should be removed

Reply:
Fixed